### ISSUE TITLE
Issue: Buttons are not disabled when writing

### ISSUE THREAD

**Issue by FlorianKempenich** [ddvk/remarkable-hacks#63]
Hi,

On the original firmware, if I'm writing the buttons are disabled. So that prevents the page from turning when my hand presses a button while I'm writing at the bottom of the page.

Would it be possible to restore that feature in your firmware? 

Best,
Florian

**Comment by ddvk** [comment]
this is a tough one, it interferes with zen mode switching when the pen is near the surface (and maybe lamy pen eraser).

**Comment by FlorianKempenich** [comment]
Oh I see. Then maybe an option to disable the buttons? Would that be a good alternative? :)

**Comment by FlorianKempenich** [comment]
Awesome. Thank you ! 😃

**Comment by FlorianKempenich** [comment]
Hey,

I'm re-opening because there's some (most likely) unintended behavior when disabling the buttons.  

When disabling the buttons it doesn't just disable the navigation buttons . . . it also disables the *button of the Lamy Pen* 😕

Would it be possible to implement a version that only disables the navigation button? That way, the writing experience would remain untouched 😃
Currently, after I disable the buttons I can't use the eraser button on my Lamy pen . . . and since that's much more annoying than accidentally pressing the navigation buttons . . . I just don't ever use the 'disable buttons' feature, but if I could use the lamy pen while the buttons are disabled, then it'd be _on_ all the time basically 🙂

Either way, thanks for your work and being so quick at reacting to feedback 😃👍

**Comment by chimpdude** [comment]
I am confused about this. 

For a few days now, I've had serious trouble getting the pen-eraser toggle to work in Zen mode, needing to press  the left button 2-3+ times to get it to switch. I just came across this thread, and now I see the issue: if the marker is close to the screen, then all functionality for the buttons is disabled. I guess it's still considered "writing" even with the marker lifted 1-2 cm off the screen? If I put the marker very far from that tablet, there is still a ~2 second delay before buttons begin working again...this almost makes the stock method of opening the menu and selecting the eraser faster than Zen mode's left-button toggle.

I don't remember this happening before. I swear I used to be able to toggle the eraser super fast, like as soon as I notice a mistake while writing I could stop writing and toggle the eraser boom-boom-bam and be back to writing again. Now it's a much slower process.

Has disabling the buttons while the marker is close to the screen plus a 2 sec delay after that always been the default behavior? Or did the stock reMarkable firmware introduce this in one of the recent updates? Or was it one of the patches that added this? Am I just insane and creating false memories??
